app/models/credit.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: credits
+#
+#  id         :integer          not null, primary key
+#  cost       :decimal(, )
+#  order_id   :integer
+#  quote_id   :integer
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class Credit < ApplicationRecord
+  belongs_to :order
+  delegate :company, to: :order
```

Comment:
cool, what does this save us from needing to do? From the docs, doesn't Credit need to have to be able to call :company in order to delegate it to an order?

Reply:
I originally did it to access the company but I just tested it without and you can still through the order. commented that out.